mimic/test/test_heat.py

```diff
@@ -159,6 +184,68 @@ def test_list_stacks_two(self):
         self.assertEqual(set(stack['stack_name'] for stack in two_stack_list),
                          set(['foostack', 'barstack']))
 
+    def test_check_stack(self):
+        """
+        Stack updates with correct check status.
+        """
+        foo_resp = self.create_stack('foostack')
+        self.check_stack('barstack', foo_resp['stack']['id'])
+        new_stack_list = self.list_stacks()['stacks']
+        self.assertEqual(new_stack_list[0]['stack_status'], 'CHECK_COMPLETE')
+
+    def test_invalid_action(self):
+        """
+        Invalid stack-action requests generate 400.
+        """
+        foo_id = self.create_stack('foo')['stack']['id']
+        self.check_stack('foo', foo_id, req_body={'foo': None}, resp_code=400)
+
+    def test_missing_action(self):
+        """
+        Invalid stack-action requests generate 400.
+        """
+        foo_id = self.create_stack('foo')['stack']['id']
+        self.check_stack('foo', foo_id, req_body={}, resp_code=400)
+
+    def test_multiple_actions(self):
+        """
+        Supplying more than one action in stack-action requests generate 400.
+        """
+        foo_id = self.create_stack('foo')['stack']['id']
+        self.check_stack('foo', foo_id,
+                         req_body={'check': None, 'cancel_update': None},
+                         resp_code=400)
+
+    def test_disabled_actions(self):
+        """
+        Stack updates with correct check status.
```

Comment:
The doc doesn't seem to match. How about "Performing unimplemented actions results in 405"